src/legacy/core_plugins/kibana/public/dashboard/panel/panel_header/panel_header.tsx

```diff
@@ -28,8 +28,8 @@ export interface PanelHeaderProps {
   title?: string;
   panelId: PanelId;
   embeddable?: Embeddable;
-  isViewOnlyMode: boolean;
-  hidePanelTitles: boolean;
+  isViewOnlyMode?: boolean;
+  hidePanelTitles?: boolean;
```

Comment:
Looks like these were previously required and are now being marked as optional?

Reply:
Thanks! Matching with what's on master and it's not causing issues.